src/constants.ts

```diff
@@ -98,6 +98,16 @@ export const PACKAGING_UNITS_CODED_CONCEPT_ID =
 export const STOCK_ITEM_CATEGORY_CONCEPT_ID =
   "6d24eb6e-b42f-4706-ab2d-ae4472161f6a";
 
+export const INVENTORY_CLERK_ROLE_UUID = "d210eb66-2188-11ed-9dff-507b9dea1806";
```

Comment:
are these uuids going to be constant for implementations ??

Reply:
Yes. This roles are constant being introduced by the backend module i.e commodity module. You can refer [https://github.com/METS-Programme/openmrs-module-ugandaemr-commodity/blob/3957f960275b3f8dcb968dfb7d6699dd8f74d68d/api/src/main/resources/stockmgmt/metadata/Role_Privilege.xml#L31](url)

Reply:
Since we depend on that module, they will remain as such. Thanks @jabahum